src/platforms/php/guides/symfony/index.mdx

```diff
@@ -19,30 +19,46 @@ composer require sentry/sentry-symfony
 
 ### Configure
 
-Add your DSN to `app/config/config.yml`:
+Add your DSN to `config/packages/sentry.yaml`:
 
-```yaml {filename:app/config/config.yml}
+```yaml {filename:config/packages/sentry.yaml}
 sentry:
   dsn: "___PUBLIC_DSN___"
 ```
 
-If you **are not** using Symfony Flex, you'll also need to enable the bundle in `app/AppKernel.php`:
+If you **are not** using Symfony Flex, you'll also need to enable the bundle in `config/bundles.php`:
 
-```php {filename:app/AppKernel.php}
+```php {filename:config/bundles.php}
```

Comment:
I'm not sure about this. Isn't the `bundles.php` file a Flex exclusive? IIRC without Flex you need to operate in the `Kernel.php` class, like shown in https://github.com/getsentry/sentry-symfony/pull/452

Reply:
`app/Kernel.php` does not longer exist in symfony 4/5. The new symfony folder structure is `config` and this is normally not depending on if you use flex or not at you would normally still follow the symfony folder structure with `config/bundles.php` when doing symfony 4/5, you just don't have the composer flex plugin which will automatically add it for you. But we could show both examples with `app/Kernel.php` (symfony 2/3)  and the symfony 4/5 structure in this documentation.

Reply:
FTR the bundle now requires 3.4 at the very least: https://github.com/getsentry/sentry-symfony/blob/d6a738f005b57a16f156f3ae1024ec874c21b2a4/composer.json#L26-L32

I would make the Flex approach more prominent, but yeah, showing the old way is ok too.

Reply:
m2c: considering that the status of [`3.4`](https://symfony.com/releases/3.4) is security fixes only, I would not bother adding any documentation for it because I find unlikely that someone will install the bundle on such an old (and "unmaintained") version. I would rather focus our efforts on `>= 4.x` versions